test-case/check-ipc-flood.sh

```diff
@@ -32,27 +32,30 @@ loop_cnt=${OPT_VALUE_lst['l']}
 
 [[ ! "$(sof-kernel-dump.sh|grep 'sof-audio'|grep 'Firmware debug build')" ]] && dlogw "${BASH_SOURCE[0]} need debug version firmware" && exit 2
 
-func_lib_setup_kernel_last_line
+func_lib_setup_kernel_last_timestamp
 func_lib_check_sudo
 
 dlogi "Check sof debug fs environment"
 [[ "$(sudo file $ipc_flood_dfs|grep 'No such file')" ]] && dlogw "${BASH_SOURCE[0]} need $ipc_flood_dfs to run the test case" && exit 2
 dlogi "Checking ipc flood test!"
 
-# cleanup dmesg buffer before test
-sudo dmesg -c > /dev/null
+# discard old kernel logs
+func_lib_setup_kernel_last_timestamp
 
 for i in $(seq 1 $loop_cnt)
 do
     dlogi "===== [$i/$loop_cnt] loop Begin ====="
     dlogc "sudo bash -c 'echo $lpc_loop_cnt > $ipc_flood_dfs'"
     sudo bash -c "'echo $lpc_loop_cnt > $ipc_flood_dfs'"
 
-    sof-kernel-log-check.sh 0 || die "Catched error in dmesg"
+    dlogc "last timestamp check: $KERNEL_LAST_TIMESTAMP"
+    sof-kernel-log-check.sh $KERNEL_LAST_TIMESTAMP || die "Catched error in journalctl"
 
     dlogi "Dumping test logs!"
-    dmesg | grep "IPC Flood count" -A 2
+    journalctl -k --since=$KERNEL_LAST_TIMESTAMP | grep "IPC Flood count" -A 2
```

Comment:
```suggestion
    journalctl -k --since="$KERNEL_LAST_TIMESTAMP" | grep "IPC Flood count" -A 2
```
KERNEL_LAST_TIMESTAMP has space inside, need quote

Reply:
> KERNEL_LAST_TIMESTAMP has space inside, need quote

@plbossart shell variable expansion needs to  be double-quoted 95% of the time and is harmless in 4.5% of the time.

Double quote first, think later. 
https://mywiki.wooledge.org/Quotes#I.27m_Too_Lazy_to_Read.2C_Just_Tell_Me_What_to_Do

Shellcheck will warn every time https://github.com/koalaman/shellcheck/wiki/SC2046

Reply:
I don't know why you are arguing, this was fixed
https://github.com/thesofproject/sof-test/pull/354/commits/8cb97f7d42e253ba179d1ad529d0d5a419fb097d#diff-7467b2c8abcf990e16a6e4f76a7a1211R54